README.md

```diff
@@ -1,9 +1,15 @@
 # leapuvc
 
-A set of examples demonstrating the usage and functionality of interfacing with the Leap Motion Peripheral via UVC.
+LeapUVC gives you access to the Leap Motion Controller image data through the industry standard UVC (Universal Video Class) interface. This gives you low level controls such as LED brightness, gamma, exposure, gain, resolution, and more.
+
+The LeapUVC release features examples in C, Python, and Matlab, as well as OpenCV bindings that show how to stream from multiple devices, track ArUco markers, change camera settings, grab lens distortion parameters, and compute stereo depth maps from the images. Use the Leap Motion Controller to track physical objects, capture high-speed infrared footage, or see the world in new ways.
+
+The bundled firmware, which is required to unlock LeapUVC access, has only been tested successfully on Windows and Linux. Mac is not currently supported. Since this is experimental software, use it at your own risk.
```

Comment:
The firmware flashing tools are Windows .exe's, but otherwise the device will recognize on all three platforms (just in magenta with the operating system's camera viewers).  The built-in examples just don't parse properly on Mac.

Reply:
Can a user install the firmware on Linux using a different flashing tool?

Reply:
No; we don't have any Linux based flashing tools unfortunately.